scripts/run-integration-tests.sh

```diff
@@ -196,6 +197,17 @@ if [[ $RUN_KOPS_TEST == true ]]; then
 fi
 ADDONS_CNI_IMAGE=$($KUBECTL_PATH describe daemonset aws-node -n kube-system | grep Image | cut -d ":" -f 2-3 | tr -d '[:space:]')
 
+if [[ $RUN_CALICO_TEST == true ]]; then
+    $KUBECTL_PATH apply -f "$TEST_CALICO_PATH"
+    sleep 5
+    while [[ $($KUBECTL_PATH describe ds calico-node -n=kube-system | grep "Available Pods: 0") ]]
+    do
+        sleep 5
+        echo "Waiting for calico daemonset update"
+    done
+    echo "Updated calico daemonset!"
+fi
```

Comment:
This just applies the Calico config. We should probably add some network policy as well, and have some custom test verifying that the policy has been applied.